pages/automatedtestingcodeceptjs.md

```diff
@@ -5,7 +5,7 @@ See [Documentations on Google Docs](https://docs.google.com/document/d/1FFpCPiTY
 ## Motivation
 While writing tests with the Python `unittest` library is awesome, writing tests in CodeceptJS is super simple and much more intuitive.
 
-## Code
+### Code
```

Comment:
Could you explain why you change all the Heading 2 in `automatedtestingcodeceptjs.md` except `Motivation` to Heading 3?